### PR TITLE
Update NATS and RabbitMQ BBEs with latest conventions

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$subject

### CODE REVIEW

File: examples/nats-service-secure-connection/nats_service_secure_connection.bal [hunk 1]
```diff
@@ -19,6 +19,7 @@ listener nats:Listener securedEP = new(nats:DEFAULT_URL,
 
 // Binds the consumer to listen to the messages published to the 'orders.valid' subject.
 service "orders.valid" on securedEP {
```
Comment (shafreenAnfar) [inline examples/nats-service-secure-connection/nats_service_secure_connection.bal:21]: `securedEP` -> `orderListener`

File: examples/rabbitmq-transaction-producer/rabbitmq_transaction_producer.bal [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
-        });
-        check commit;
+    function init() returns error? {
+        // Initiate the RabbitMQ client at the start of the service. This will be used
+        // throughout the lifetime of the service.
+        self.orderClient = check new (rabbitmq:DEFAULT_HOST, rabbitmq:DEFAULT_PORT);
+    }
+
+    resource function post orders(@http:Payload Order newOrder) returns http:Accepted|error {
+
```
Comment (shafreenAnfar) [inline examples/rabbitmq-transaction-producer/rabbitmq_transaction_producer.bal:21]: We don't need the new line here.

File: examples/rabbitmq-transaction-producer/rabbitmq_transaction_producer.bal [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
 import ballerinax/rabbitmq;
 
-public type Order record {|
+type Order readonly & record {
     int orderId;
     string productName;
     decimal price;
     boolean isValid;
-|};
+};
```
Comment (ThisaruGuruge) [inline examples/rabbitmq-transaction-producer/rabbitmq_transaction_producer.bal:9]: Shouldn't this be a closed record?

File: examples/rabbitmq-client-secure-connection/rabbitmq_client_secure_connection.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # RabbitMQ client - SSL/TLS
 
-The `rabbitmq:Client` can be configured to communicate through HTTPS by providing a certificate file. The certificate can be provided through the `secureSocket` field of the connection configuration. Use this to secure the communication between the client and the server.
+The `rabbitmq:Client` can be configured to connect to the server via SSL/TLS by providing a certificate file. The certificate can be provided through the `secureSocket` field of the connection configuration. Use this to secure the communication between the client and the server.
```
Comment (ThisaruGuruge) [inline examples/rabbitmq-client-secure-connection/rabbitmq_client_secure_connection.md:3]:
```suggestion
The `rabbitmq:Client` can be configured to connect to the server via SSL/TLS by providing a certificate file. The certificate can be provided through the `secureSocket` field of the `rabbitmq:ConnectionConfiguration`. Use this to secure the communication between the client and the server.
```

File: examples/nats-service-secure-connection/nats_service_secure_connection.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # NATS service - SSL/TLS
 
-The `nats:Listener` can be configured to communicate through HTTPS by providing a certificate file. The certificate can be provided through the `secureSocket` field of the connection configuration. Use this to secure the communication between the client and the server.
+The `nats:Listener` can be configured to connect to the server via SSL/TLS by providing a certificate file. The certificate can be provided through the `secureSocket` field of the connection configuration. Use this to secure the communication between the client and the server.
```
Comment (ThisaruGuruge) [inline examples/nats-service-secure-connection/nats_service_secure_connection.md:3]:
```suggestion
The `nats:Listener` can be configured to connect to the server via SSL/TLS by providing a certificate file. The certificate can be provided through the `secureSocket` field of the `nats:ConnectionConfiguration`. Use this to secure the communication between the client and the server.
```

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**